src/log/env_config.cr

```diff
@@ -0,0 +1,17 @@
+class Log
+  # Setups *builder* based on `CRYSTAL_LOG_LEVEL` and `CRYSTAL_LOG_SOURCES`
+  # environment variables.
+  def self.setup_from_env(*, builder : Log::Builder = Log.builder,
```

Comment:
Shouldn't that be called `setup` with a `NOTE` in docs about using `ENV` as default argument values?
I mean there's nothing in the method itself related to the `ENV`, aside of the before-mentioned default values...

Reply:
Is there another way to setup the log configuration?  It seems weird to have to pass strings into the config API.

Reply:
I think calling it setup_from_env is more clear regarding what it does. But no strong feelings, more a evolution of the code regarding env vs yaml. 

@straight-shoota any preferences?

Reply:
This is not a *generic* setup method, but a specific implementation based on confguration values from env. So I agree that the current name fits the behaviour.

Reply:
@straight-shoota I disagree. The implementation is totally generic, only relation to the `ENV` is via argument defaults. Also, @wontruefree raised a valid issue - how would you setup logging in a generic manner?

The way it is right now:

```cr
Log.setup_from_env(
  backend: Log::IOBackend.new(STDOUT),
  level: "INFO"
)
```

Makes lil' sense to me, logic-wise (no `ENV` relation at all here despite using `setup_from_env` method).

Reply:
I don't follow. Why would you want to use this a method in the first place? It's very specific for setting up logging from environment variables. That's it's sole purpose. If you want something else (for example hard coding the logger setup), there's no point in using this method.

This is a much simpler way to set things up as in your example. No need for calling `#setup_from_env`.
```cr
Log.builder.bind("", :info, Log::IO::Backend.new(STDOUT))
```